common/storage/sql/sqlite3/tests/test_sqlite3.c

```diff
@@ -73,6 +74,28 @@ void test_stored_transaction(void) {
   }
 }
 
+void test_stored_load_hashes_by_address(void) {
+  trit_array_p hashes[5];
+
+  for (int i = 0; i < 5; ++i) {
+    hashes[i] = trit_array_new(FLEX_TRIT_SIZE_243);
```

Comment:
Shouldn't it be just 243 here (not flex) ?
`trit_array_new` already takes care of the conversion.

Reply:
That is correct, thanks